website/components/Home/CallToAction/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import classNames from 'classnames';
+
+import styles from './CallToAction.module.css';
+
+const Button = ({ title, outlined, large, primary }) => (
+  <span
+    className={classNames(
+      styles.button,
+      outlined && styles.outlined,
+      large && styles.large,
+      primary && styles.primary
+    )}
+  >
+    {title}
+  </span>
+);
+
+const Section = ({ children, style, className }) => (
+  <section style={style} className={classNames(styles.container, className)}>
+    {children}
+  </section>
+);
+
+export function SectionButton({ title, vertical }) {
+  return (
+    <Section
+      style={{
+        marginTop: '-3rem',
+        marginBottom: '4rem'
+      }}
```

Comment:
Why these two styles are inline?

Reply:
Because otherwise I would have to put them in separate class but these are only needed in this specific single situation.